src/core/pyproxy.ts

```diff
@@ -2035,7 +2050,11 @@ const PyProxyHandlers = {
     // Note: must report "prototype" in proxy when we are callable.
     // (We can return the wrong value from "get" handler though.)
     let objHasKey = Reflect.has(jsobj, jskey);
-    if (objHasKey) {
+    if (
+      objHasKey &&
+      jskey in jsobj &&
```

Comment:
isn't this exactly what `Reflect.has(jsobj, jskey)` returns?

Reply:
Oops good point.